recipes-graphics/raspidmx/raspidmx_git.bb

```diff
@@ -0,0 +1,36 @@
+SUMMARY = "A set of DispmanX examples"
+HOMEPAGE = "https://github.com/AndrewFromMelbourne/raspidmx"
+SECTION = "graphics"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=52962875ab02c36df6cde47b1f463024"
+
+COMPATIBLE_HOST_rpi = "${@bb.utils.contains('MACHINE_FEATURES', 'vc4graphics', 'null', '(.*)', d)}"
+
+SRC_URI = "git://github.com/AndrewFromMelbourne/raspidmx;protocol=https \
+           file://0001-gitignore-add-archives-from-lib-directory.patch \
+           file://0002-add-install-targets-to-Makefiles.patch \
+           file://0003-switch-to-pkg-config.patch \
+           file://0004-add-libvchostif-to-link.patch \
+           file://0005-change-library-linking-order.patch \
+           file://0006-game-Makefile-install-sample-png-files.patch \
+           file://0007-Makefile-reorganize.patch \
+           "
+
+PV = "1.0+git${SRCPV}"
```

Comment:
Any reason to set this to 1.0? I don't see that version in upstream and it might be a bit confusing.

Reply:
Agreed. It is a leftover from "devtool add…".
What do you recommend? Removing it? Setting it to "git${SRCPV}"? Something else?

Reply:
That or "0.0+git${SRCPV}". I saw that is used in other components in poky.

Reply:
Sounds good. I've re-pushed these two changes.